extension/persistence/relational-jdbc/src/main/java/org/apache/polaris/extension/persistence/relational/jdbc/JdbcMetaStoreManagerFactory.java

```diff
@@ -106,12 +111,21 @@ private void initializeForRealm(
   }
 
   private DatasourceOperations getDatasourceOperations(boolean isBootstrap) {
-    DatasourceOperations databaseOperations = new DatasourceOperations(dataSource);
+    // TODO: remove when multiple dataSources are supported.
+    if (dataSources.size() > 1) {
+      throw new IllegalStateException("More than one dataSources configured");
+    }
+    DatasourceOperations databaseOperations = new DatasourceOperations(dataSources.getFirst());
     if (isBootstrap) {
```

Comment:
Just to recap: I believe it is preferable to run DDL from the dedicated `bootstrap` command or higher-level class and avoid conditional DDL yes/no logic on every call to `getDatasourceOperations` (for follow-up).